arch/arm/boot/dts/qcom/msm8939.dtsi

```diff
@@ -27,7 +27,7 @@
 	qcom,msm-id = <239 0>, <241 0>, <263 0>;
 
 	chosen {
-		bootargs = "boot_cpus=0,4,5,6,7 sched_enable_hmp=1";
+		bootargs = "boot_cpus=0,1,2,3,4,5,6,7 sched_enable_hmp=1";
```

Comment:
Should we enable all cores on boot? Maybe leave core 1, and the rest (0,2,3,4,5,6,7) to boot?